samples/net/full_mqtt_tls/src/private_info/create_keys.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 Linaro
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import os
+import re
+import subprocess
+import argparse
+
+def parse_args():
+    global args
+
+    parser = argparse.ArgumentParser(description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    parser.add_argument(
+        "-d", "--device", required=True,
+        help="Name of device")
+
+    parser.add_argument(
+        "-e", "--ecdsa", required=False,
+        action='store_true',
+        help="Use elliptic curve")
+
+    parser.add_argument(
+        "-r", "--rsa", required=False,
+        action='store_true',
+        help="Use RSA")
+
+    args = parser.parse_args()
+
+split_string = lambda x, n: [x[i:i+n] for i in range(0, len(x), n)]
+
+def main():
+    parse_args()
+
+    fd = open("key.c", "w")
+
+    pem_file = args.device + "-private.pem"
+    cert_file = args.device + "-cert.pem"
+    der_file = args.device + "-private.der"
+
+    if args.ecdsa:
+        print("Generating ecdsa private key")
+        subprocess.call(["openssl", "ecparam", "-noout", "-name", "prime256v1", "-genkey",
+            "-out", pem_file])
+    else:
+        print("Generating rsa private key")
+        subprocess.call(["openssl", "genrsa", "-out", pem_file, "2048"])
+
+    print("Generating certificate")
+    subprocess.call(["openssl", "req", "-new", "-x509", "-key", pem_file, "-out",
```

Comment:
Should use subprocess.check_call so that errors aren't ignored. Likewise other uses in this file.

Reply:
fixed all instances.  used try except blocks